File: common-core/src/main/java/com/ityu/dao/system/NoticeRepository.java
package com.ityu.dao.system;



import com.ityu.bean.entity.system.Notice;
import com.ityu.dao.BaseRepository;

import java.util.List;

/**
 * Created  on 2018/3/21 0021.
 *
 * @author enilu
 */
public interface NoticeRepository extends BaseRepository<Notice,Long> {
    List<Notice> findByTitleLike(String name);
}

File: common-core/src/main/java/com/ityu/common/dao/system/NoticeRepository.kt
package com.ityu.common.dao.system

import com.ityu.common.bean.entity.system.Notice
import com.ityu.common.dao.BaseRepository

/**
 * Created  on 2018/3/21 0021.
 *
 * @author enilu
 */
interface NoticeRepository : BaseRepository<Notice?, Long?> {
    fun findByTitleLike(name: String?): List<Notice?>?
}
